libs/pika/pack_traversal/tests/unit/pack_traversal.cpp

```diff
@@ -61,12 +61,12 @@ struct all_map
 static void test_mixed_traversal()
 {
     {
-        auto res = map_pack(all_map_float{}, 0, 1.f, pika::make_tuple(1.f, 3),
+        auto res = map_pack(all_map_float{}, 0, 1.f, std::make_tuple(1.f, 3),
             std::vector<std::vector<int>>{{1, 2}, {4, 5}},
             std::vector<std::vector<float>>{{1.f, 2.f}, {4.f, 5.f}}, 2);
 
-        auto expected = pika::make_tuple(    // ...
-            1.f, 2.f, pika::make_tuple(2.f, 4.f),
+        auto expected = std::make_tuple(    // ...
+            1.f, 2.f, std::make_tuple(2.f, 4.f),
```

Comment:
Do we need the `std::` namespace since there is the `using` statement at the beginning of the file?